src/content/docs/frameworks/Asp.netCore/AspNetCore-case-study.md

```diff
@@ -0,0 +1,316 @@
+  
+- It also provides model validation, making it simple to validate the request, and provide quick error messages if needed.     
+- **Supports dependency injection:** Provides an IOC container that you can register services with, and use to inject your dependency at runtime. Abstractions make it so you only need to register the service and its life time. Then you can add the service either by class constructor or method injection.  
+- **Natively supports the MVC pattern:** Asp.net Core provides an opinionated way of using the mvc pattern, having predefined interfaces and classes to implement or inherit from to streamline the process. 
+- **Attribute routing:** You can create your routes directly on your action methods inside your  controllers making development process faster, and gives you an better overview of what action methods are coupled with what endpoints.
+
+
+## Market Comparison
+
+Instead of having and extensive comparison how asp.net core compares to many of the popular frameworks like Laravel, Spring, Django and Express.js. I have decided to solely compare it to express.js since i expect that to be the most likely backend framework front end developers are familiar with. 
+
+| Feature                | ASP.NET Core                                     | Express.js                                     |
+|------------------------|--------------------------------------------------|------------------------------------------------|
+| **Language**           | C#                                               | JavaScript/TypeScript                          |
+| **Runtime**            | .NET                                             | Node.js                                        |
+| **Ecosystem**          | .NET ecosystem, Visual Studio, Entity Framework | npm, vast JavaScript ecosystem                 |
+| **Cross-Platform**     | Yes (Windows, Linux, macOS)                      | Yes (any Node.js-supported platform)           |
+| **Ease of Use**        | Structured, more features out-of-the-box        | Simple, minimalistic, flexible                 |
+| **Learning Curve**     | Steeper (especially without .NET background)    | Gentler (especially for those familiar with JS)|
+| **Use Cases**          | Wide range (from small websites to enterprise applications) | RESTful APIs, SPAs, real-time applications |
+| **Development Model**  | More structured, feature-rich                   | Minimalistic, highly flexible                  |
+| **Community Support**  | Strong (backed by Microsoft)                     | Extremely large and active                     |
+| **Typing**             | Statically typed                                | Dynamically typed (TypeScript for static typing)|
+
+
+
+
+## Example of how you can build with ASP.NET CORE 
+
+-  I recommend first installing visual studio(not to be confused with visual studio code) which is an ide with good support for .net. 
```

Comment:
Using the word `I` in this article makes it sound subjective. Avoid it totally because the article should sounds objective. 

Go through the whole write-up and fix that.

Reply:
As far as I can tell, I have fixed the issue you mentioned now. Thanks for great feedback!